### DISCUSSION TITLE
help me with this errors

### DISCUSSION THREAD

**Discussion by therealtinhtute** [spencerwooo/onedrive-vercel-index#104] [Q&A] [answered]
`[GET] /api?path=/
21:56:05:44
Function Status:
None
Edge Status:
500
Duration:
433.50 ms
Init Duration:
N/A
Memory Used:
33 MB
ID:
gqwsn-1631112965440-5596d5e47bff
User Agent:
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/86.0.4240.198 Safari/537.36
0.AXEA7X7I8kIT10-qNYGogbma_jlstW1IV51LjWj3USGP3AVxAPU.AgABAAAAAAD--DLA3VO7QrddgJg7WevrAgDs_wQA9P_bA8npFpJP51Flh60Lzubuwb1haPZ8cOf9n0OwYejsu9U9p4_SyQ9pNCt1f2zikDxt-GfgwMr63cOO0PYetVAoi0w2eoyh9piTCy5I-vnROxfY7-gBhalPzlrFZpAQKmlwclEiWG5kytk9L_MqrZGdym7IniIEwGvMte1TADi7LOCCALtQ2P93mc9NtPS_GczOE9N6vEwnaFSpa623jB5Bzs_fPqxRWy0QAy3OUkgCls3_UWgqMXnQDXz5YLNW9mQFG47nrJChdcnWE9ui60ZAL_f-Wesf6JqxMla0IvCpQscqhAqTiUTE7jmye71HLSTjfBG1hhzkz0sHY97E31Sn-0NWiafGRWvmmshyLtvBFoxtQ3IVBcFJBdCqboCH5tMCbc0iRN5iHs2-TN6I3NRq7jTYKMur2yiRla2rL2KKkPt5pmwg6ZSjybI67V2VawBsAfmOFGBRJn6H2GjkuUAO12_v1ZmTUxvQAJ79x4YNDJchLvD4GVV0VG1OV_zDYRWxU0TzZttBQSUr5xFBLyAYqdOrPmeIMog1rYYKyyfn4sW6C2zNTBUnFxd_V1F94tITvxCT32d-jEt16RBscrKiS0n0gL8w87tcG1ECFYvwuSl-iONkCqLwSbK1UISpFjsbYL4DF0udgfhnGaqlwh5NT3Yw1EwvaJ2bFwJdB-O_eQI1j2h_Sn7DfiVXwsof_O2bja5YEP-J0fmsiLCAh82rHiMph7Dct5S0ojbU2v531tZOvZyrh4ge5mcIOqk-LoiLPE9oW4AESdmBFqjfzRdzd1pCCcDUyNqbJRsfpSCszdByy63Js2_WlV0CRee-Gua4dBTsxaTgAerGODmaWkvoUpHmPuxDjGKsQ6amyKKxBfgVfPTLZ4POtTWkeloSwQ6gJXCSY6zkrpSwBuEp6i1fEJ41pnb65Z9ni1XjjYvw8ucjFES2JrOhiAu8DghxiJ7t-OKUQSfsI_qcLnxa3TK1KD08855pX78&grant_type=refresh_token',
      headers: [Object],
      transformRequest: [Array],
      transformResponse: [Array],
      timeout: 0,
      adapter: [Function: httpAdapter],
      xsrfCookieName: 'XSRF-TOKEN',
      xsrfHeaderName: 'X-XSRF-TOKEN',
      maxContentLength: -1,
      maxBodyLength: -1,
      validateStatus: [Function: validateStatus]
    },
    request: <ref *1> ClientRequest {
      _events: [Object: null prototype],
      _eventsCount: 7,
      _maxListeners: undefined,
      outputData: [],
      outputSize: 0,
      writable: true,
      destroyed: false,
      _last: true,
      chunkedEncoding: false,
      shouldKeepAlive: false,
      _defaultKeepAlive: true,
      useChunkedEncodingByDefault: true,
      sendDate: false,
      _removedConnection: false,
      _removedContLen: false,
      _removedTE: false,
      _contentLength: null,
      _hasBody: true,
      _trailer: '',
      finished: true,
      _headerSent: true,
      socket: [TLSSocket],
      _header: 'POST /common/oauth2/v2.0/token HTTP/1.1\r\n' +
        'Accept: application/json, text/plain, */*\r\n' +
        'Content-Type: application/x-www-form-urlencoded\r\n' +
        'User-Agent: axios/0.21.1\r\n' +
        'Content-Length: 1261\r\n' +
        'Host: login.microsoftonline.com\r\n' +
        'Connection: close\r\n' +
        '\r\n',
      _keepAliveTimeout: 0,
      _onPendingData: [Function: noopPendingOutput],
      agent: [Agent],
      socketPath: undefined,
      method: 'POST',
      maxHeaderSize: undefined,
      insecureHTTPParser: undefined,
      path: '/common/oauth2/v2.0/token',
      _ended: true,
      res: [IncomingMessage],
      aborted: false,
      timeoutCb: null,
      upgradeOrConnect: false,
      parser: null,
      maxHeadersCount: null,
      reusedSocket: false,
      host: 'login.microsoftonline.com',
      protocol: 'https:',
      _redirectable: [Writable],
      [Symbol(kCapture)]: false,
      [Symbol(kNeedDrain)]: false,
      [Symbol(corked)]: 0,
      [Symbol(kOutHeaders)]: [Object: null prototype]
    },
    data: {
      error: 'invalid_grant',
      error_description: 'AADSTS70000: Provided grant is invalid or malformed.\r\n' +
        'Trace ID: f9a2dcea-3962-45c4-bbf3-c7a2572afa00\r\n' +
        'Correlation ID: 189787d7-5039-474e-9ae7-727e9adb5727\r\n' +
        'Timestamp: 2021-09-08 14:56:05Z',
      error_codes: [Array],
      timestamp: '2021-09-08 14:56:05Z',
      trace_id: 'f9a2dcea-3962-45c4-bbf3-c7a2572afa00',
      correlation_id: '189787d7-5039-474e-9ae7-727e9adb5727',
      error_uri: 'https://login.microsoftonline.com/error?code=70000'
    }
  },
  isAxiosError: true,
  toJSON: [Function: toJSON]
}
RequestId: 7337b0a5-569e-49e6-8a4c-c3453ea2d91b Error: Runtime exited with error: exit status 1
Runtime.ExitError`
出现此错误时，我正在 Vercel 上进行安装。 请帮帮我。

**Comment by Divkix** [comment]
Is this after latest update?
if yes, then try deploying from an older commit
I'm too getting this issue, however [this](https://github.com/spencerwooo/onedrive-vercel-index/commit/97f4250a57985407c164a2cda6d6c088f0453665) works without any issues

**Comment by spencerwooo** [comment, marked as answer]
PR #105 seems to have solved this problem, try syncing changes to the latest version please.
